cmake/modules/CheckLinkerDynamicRelocationFlags.cmake

```diff
@@ -0,0 +1,11 @@
+include(CheckCXXCompilerFlag)
+
+function(check_linker_supports_flag flag_to_test var_name)
+  set(CMAKE_REQUIRED_FLAGS "${flag_to_test}")
+  check_cxx_compiler_flag("" LINKER_SUPPORTS_NOEXTERN_PROTECTED_DATA)
+  if(LINKER_SUPPORTS_NOEXTERN_PROTECTED_DATA)
```

Comment:
(I could have swore I replied to this already but apparently not.)

If I change it to `result` than the check in the CMake output is less meaningful. It displays something like

```
...
Performing Test result
Performing Test result - Success
...
```

as opposed to

```
...
Performing Test LINKER_SUPPORTS_NOEXTERN_PROTECTED_DATA
Performing Test LINKER_SUPPORTS_NOEXTERN_PROTECTED_DATA - Success
...
```

If you still think it is worth it than I'm amenable to the change but I think it provides a less clear description of what is going on.